### ISSUE TITLE
Proposal: Reduce the height of a menu item when there is no subtitle

### ISSUE THREAD

**Issue by mGeek** [pebble/pebblejs#68]
Hey there,

Would it be possible to reduce the menu items when there is no subtitle ?

Here is the Pebble setup menu :
![empghm8qinwaaaaasuvork5cyii](https://cloud.githubusercontent.com/assets/1749532/8208068/b07beac4-1505-11e5-982f-22a0e6fe0263.png)

Here is a screenshot of my application, with no subtitles :
![xzueynqmgvpibzdukea9b l9are5ae2eaaaaabjru5erkjggg](https://cloud.githubusercontent.com/assets/1749532/8208071/b486094c-1505-11e5-9aad-45aa4fd42a76.png)

What do you think about that ?

**Comment by Meiguro** [comment]
I agree, the menu should definitely be more flexible. There needs to be a way to customize the look and feel of a menu cell. Eventually I would like a menu cell to be as dynamic as the dynamic window, but a good API for that would take awhile to design and implement.

The reason why the menu maintains the height regardless of whether there is a subtitle or not is to reduce jumping. Jumping can be caused by menu items that dynamically change or by menu items becoming loaded or unloaded.

In the mean time, we can try to think of some interim API that is most likely to be a smaller part of what a future API would be. Maybe a menu layer can have a default cell layout to be specified. We can call the current cell layout to be Basic. What you're proposing is something like Thin or Small.

Would you need the layout to automatically size based on the subtitle, or does setting the size of all cells to be the same satisfy your use case?

**Comment by fletchto99** [comment]
I suggest we could use [menu_cell_title_draw](http://developer.getpebble.com/docs/c/User_Interface/Layers/MenuLayer/#menu_cell_title_draw) in the draw row callback if no subtitle is present. I'll implement this tonight and see how it looks. It should be a quick and easy fix until something more robust is added in the future.

**Comment by compelling** [comment]
Hi, especially on Time Round, the single line menu lines look very high. Users of my pebble.js based grocery app have asked for more lines/lower menu line height like it's possible in the native sdk:  https://forums.pebble.com/t/watchapp-grosh-the-intelligent-grocery-app-now-for-pebble-round-and-with-dictation/20930/2
I was traversing the docs to find ways to decrease the menu height, but this is still not implemented, or did you sneak in something undocumented a year ago when this topic was up?
Would love to have more lines in our menu-heavy grocery app!!!
Kind regards Søren, compelling software
